src/data/vapor-pressure-constants.ts

```diff
@@ -18,6 +18,308 @@ export const DIPPRVaporPressureDictionary: Record<
+    CAS: '75-15-0.',
+    C1: 67.114,
+    C2: -4820.4,
+    C3: -7.5303,
+    C4: 0.0091695,
+    C5: 1,
+    minimumTemperature: 161.11,
+    maximumTemperature: 552,
+    pressureAtMinimumTemperature: 1.49,
+    pressureAtMaximumTemperature: 8.041e6,
+  },
+
+  CarbonMonoxide: {
+    name: 'Carbon monoxide',
+    formula: 'CO',
+    CAS: '630-08-0',
+    C1: 45.698,
+    C2: -1076.6,
+    C3: -4.8814,
+    C4: 0.000075673,
+    C5: 2,
+    minimumTemperature: 68.15,
+    maximumTemperature: 132.92,
+    pressureAtMinimumTemperature: 1.54e4,
+    pressureAtMaximumTemperature: 3.494e6,
+  },
+
+  CarbonTetraChloride: {
+    name: 'Carbon tetrachloride',
+    formula: 'CCl4',
+    CAS: '56-23-5',
+    C1: 78.441,
+    C2: -6128.1,
+    C3: -8.5766,
+    C4: 6.8465e-6,
+    C5: 2,
+    minimumTemperature: 250.33,
+    maximumTemperature: 556.35,
+    pressureAtMinimumTemperature: 1.12e3,
+    pressureAtMaximumTemperature: 4.544e6,
+  },
+
+  CarbonTetraFlouoride: {
+    name: 'Carbon tetraflouride',
+    formula: 'CF4',
+    CAS: '75-73-0',
+    C1: 61.89,
+    C2: -2296.3,
+    C3: -7.086,
+    C4: 0.000034687,
+    C5: 2,
+    minimumTemperature: 89.56,
+    maximumTemperature: 227.51,
+    pressureAtMinimumTemperature: 1.08e2,
+    pressureAtMaximumTemperature: 3.742e6,
+  },
+
+  Chlorine: {
+    name: 'Chlorine',
+    formula: 'Cl2',
+    CAS: '7782-50-5',
+    C1: 71.334,
+    C2: -3855,
+    C3: -8.5171,
+    C4: 0.012378,
+    C5: 1,
+    minimumTemperature: 172.12,
+    maximumTemperature: 417.15,
+    pressureAtMinimumTemperature: 1.37e3,
+    pressureAtMaximumTemperature: 7.793e6,
+  },
+
+  Chlorobenzene: {
+    name: 'Chlorobenzene',
+    formula: 'C6H5Cl',
+    CAS: '108-90-5',
+    C1: 54.144,
+    C2: -6244.4,
+    C3: -4.5343,
+    C4: 4.7e-18,
+    C5: 6,
+    minimumTemperature: 227.95,
+    maximumTemperature: 632.35,
+    pressureAtMinimumTemperature: 8.45,
+    pressureAtMaximumTemperature: 4.529e6,
+  },
+
+  Chloroethane: {
+    name: 'Chloroethane',
+    formula: 'C2H5Cl',
+    CAS: '75-00-3',
+    C1: 44.677,
+    C2: -4026,
+    C3: -3.371,
+    C4: 2.27e-17,
+    C5: 6,
+    minimumTemperature: 136.75,
+    maximumTemperature: 460.35,
+    pressureAtMinimumTemperature: 2.61e-1,
+    pressureAtMaximumTemperature: 5.267e6,
+  },
+
+  Chloroform: {
+    name: 'Chloroform',
+    formula: 'CHCl3',
+    CAS: '67-66-3',
+    C1: 146.43,
+    C2: -7792.3,
+    C3: -20.614,
+    C4: 0.024578,
+    C5: 1,
+    minimumTemperature: 207.15,
+    maximumTemperature: 536.4,
+    pressureAtMinimumTemperature: 5.25e1,
+    pressureAtMaximumTemperature: 5.554e6,
+  },
+
+  Chloromethane: {
+    name: 'Chloromethane',
+    formula: 'CH3Cl',
+    CAS: '74-87-3',
+    C1: 44.555,
+    C2: -3521.3,
+    C3: -3.4258,
+    C4: 5.63e-17,
+    C5: 6,
+    minimumTemperature: 175.45,
+    maximumTemperature: 416.25,
+    pressureAtMinimumTemperature: 8.84e2,
+    pressureAtMaximumTemperature: 6.759e6,
+  },
+
+  Chloropropane1: {
```

Comment:
```suggestion
  `1Chloropropane`: {
```

Rename past entries also using this format.

Reply:
Okay sir